hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/bootstrap/BootstrapOperator.java

```diff
@@ -155,6 +155,7 @@ protected void preLoadIndexRecords() throws Exception {
 
     // wait for the other bootstrap tasks finish bootstrapping.
     waitForBootstrapReady(getRuntimeContext().getIndexOfThisSubtask());
+    hoodieTable = null;
   }
```

Comment:
Is there any chance we got NPE?

Reply:
hoodieTable is only used in the `preLoadIndexRecords`

Reply:
sure, can you add some comment to elaborate the details.